js/script.js

```diff
@@ -316,7 +316,11 @@ function stop_teleprompter()
 // Update Teleprompter
 function update_teleprompter()
 {
-	$.cookie('teleprompter_text', $('#teleprompter').html());
+	if(evt.keyCode == 27) { // escape
```

Comment:
There is no `evt` param being passed into this function, so not sure how this condition could ever be true.

Reply:
You're right, that's what I get for trying to copy this out of my actual working copy!